conf/test_full.config

```diff
@@ -0,0 +1,31 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Nextflow config file for running minimal tests
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Defines input files and everything required to run a fast and simple pipeline test.
+
+    Use as follows:
+        nextflow run nf-core/variantmtb -profile test,<docker/singularity> --outdir <OUTDIR>
+
+----------------------------------------------------------------------------------------
+*/
+
+params {
+    config_profile_name        = 'Full test profile'
+    config_profile_description = 'Minimal test dataset to check pipeline function. This is the same as test.config. As of now, there is no "full" test possible due to access tokens.'
+
+    // Limit resources so that this can run on GitHub Actions
+    max_cpus   = 2
+    max_memory = '6.GB'
+    max_time   = '6.h'
+
+    // Input data
+    input  = "${projectDir}/tests/csv/input.csv"
+
+    // Genome references
+    genome = 'GRCh37'
+    fasta = "s3://ngi-igenomes/igenomes/Homo_sapiens/Ensembl/GRCh37/Sequence/WholeGenomeFasta/genome.fa"
+
+    // mandatory flags
+    databases = 'civic'
+}
```

Comment:
Was unsure what a good practice here is. I have plenty smaller test cases locally but thought this would flood the config dir and make it confusing / hard to get an overview.
Also, I thought for a user the minimal test run (can be run without any prerequisites) and the full test run (covering as much of the pipeline as possible) are sufficient cases.
But those are just my 2cents, I can easily add more fine-grained test configs